avalon_client/avalon_client_sdk/interfaces/worker_registry_client.py

```diff
@@ -15,7 +15,7 @@
 from abc import ABC, abstractmethod
 
 
-class WorkerRegistryInterface(ABC):
+class WorkerRegistryClient(ABC):
     """
     WorkerRegistryInterface is an abstract base class containing abstract APIs
```

Comment:
WorkerRegistryInterface is an  -> This is an

Reply:
Done